optuna/trial/_frozen.py

```diff
@@ -129,11 +132,18 @@ def __init__(
         system_attrs: Dict[str, Any],
         intermediate_values: Dict[int, float],
         trial_id: int,
+        values: Optional[Sequence[float]] = None,
```

Comment:
I'm wondering if we shouldn't deprecate `value` in `FrozenTrial.__init__` and `create_trial`. The former is (hopefully) not that widely used and the latter is still experimental.

Reply:
Somewhat related to https://github.com/optuna/optuna/pull/1994#discussion_r540749401.

Reply:
Sure. I've fixed it.